src/routes/wishlistPage/wishlist-page.jsx

```diff
@@ -0,0 +1,34 @@
+import { ProductCard } from "../../components";
+import { useWishlist } from "../../context/wishlist-context";
+import undrawImage from "../../assets/images/walking.svg";
+import "./wishlistPage.css";
+import { useNavigate } from "react-router-dom";
+
+export const WishListPage = () => {
+  const { wishList } = useWishlist();
+  const navigate = useNavigate();
+    const goToShop = () => {
+        navigate("/shop")
+    }
+  return (
+    <>
+      {wishList.length === 0 ? <h1>WishList is empty!!</h1> : <h1>WishList</h1>}
+      <div className="wishlist-page-container">
+        {wishList.length === 0 ? (
+          <img
+            className="img-responsive img-wishlist"
+            src={undrawImage}
+            alt="Svg image"
```

Comment:
Provide a better alt description. It is a given that it is an image. Provide the purpose or the info related to it. Eg: productCard

Reply:
okay.